examples/platforms/nrf528xx/src/radio.c

```diff
@@ -160,6 +163,12 @@ static void dataInit(void)
         sReceivedFrames[i].mPsdu = NULL;
     }
 
+    for (size_t i = 0; i < OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MAX - OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MIN + 1; i++)
```

Comment:
Since `sMaxTxPowerTable` is a fixed size array, how about
```suggestion
    for (size_t i = 0; i < sizeof(sMaxTxPowerTable); i++)
```